apps/datahub/src/app/home/search/search-summary/search-summary.component.html

```diff
@@ -1,20 +1,51 @@
-<div>
-  <gn-ui-results-hits></gn-ui-results-hits>
-</div>
-<div class="mt-4" *ngIf="publisher$ | async as publisher">
-  <div translate="">search.filters.active</div>
-  <div
-    class="inline-block bg-primary py-2 px-3 rounded font-medium text-gray-50 text-sm leading-none mt-2"
-  >
-    <div class="flex">
-      <div
-        class="close bg-white bg-opacity-50 text-primary w-4 cursor-pointer h-4 text-xl leading-3 hover:bg-opacity-100 rounded-full text-center transition-all"
-        style="line-height: 14px"
-        (click)="removeOrg()"
+<div
+  class="px-3 pt-6 sm:p-7 sm:my-8 sm:shadow-primary-light sm:rounded-xl grid grid-cols-3 gap-4 transition-[height] duration-100"
+  [ngClass]="
+    isOpen ? 'bg-primary text-white h-[204px]' : 'bg-white text-main h-[98px]'
+  "
+>
+  <div class="col-span-2 grid grid-cols-2 gap-4 grid-rows-auto">
+    <div class="col-span-2 flex flex-row items-baseline" *ngIf="isOpen">
+      <h2 class="text-2xl mr-4" translate>search.filters.title</h2>
+      <gn-ui-results-hits class="text-sm opacity-[60%]"></gn-ui-results-hits>
+    </div>
+    <gn-ui-filter-dropdown
```

Comment:
Selecting values scrolls up the page, which detaches the dropdown selection position from the input.

Reply:
Good point... will try to fix this

Reply:
Ok so I think we'll need #344 for this because by default the cdk-overlay reposition strategy listens to scroll events on the document, but currently the scroll happens inside the home page. I'm leaving this like that for now.